app-center-webapps/src/main/webapp/vue-apps/userSetup/components/UserFavoriteApplications.vue

```diff
@@ -229,6 +229,12 @@ export default {
             app => app.id === appId
           );
           this.$parent.$children[0].authorizedApplicationsList[index].favorite = false;
+        }).finally(() => {
+          // make sure to hide tooltips.
+          const tooltips = document.getElementsByClassName('tooltip fade bottom');
```

Comment:
Tooltip should fade automatically, we should not add Javascript for doing that. Could you find why it is not fading automatically

Reply:
Because the component on which is added a tooltip is removed once the remove from favorite applications button is clicked.